autorest/adal/persist.go

```diff
@@ -25,6 +25,24 @@ func LoadToken(path string) (*Token, error) {
 	return &token, nil
 }
 
+// LoadCLITokens restores a set of AzureCLIToken objects from a file located at 'path'.
+func LoadCLITokens(path string) (*[]AzureCLIToken, error) {
```

Comment:
Any reason you're returning a pointer to the array here? The actual object that would be copied in memory would be pretty small.